pkg/disttask/framework/proto/subtask.go

```diff
@@ -80,10 +90,10 @@ type Subtask struct {
 	// StartTime is the time when the subtask is started.
 	// it's 0 if it hasn't started yet.
 	StartTime time.Time
-	// UpdateTime is the time when the subtask is updated.
+	// StateUpdateTime is the time when the subtask state is updated.
 	// it can be used as subtask end time if the subtask is finished.
 	// it's 0 if it hasn't started yet.
-	UpdateTime time.Time
```

Comment:
previous is fine, the column name is restricted, we might change other fields later

Reply:
Jiajia means that the subsequent `state_update_time` in table may be renamed as `update_time`, with its meaning expanded to include update time of all fields, not just state update time. What do you think? @ywqzzy @tangenta

Reply:
I prefer `update_time` as well.

Reply:
> Jiajia means that the subsequent state_update_time in table may be renamed as update_time

i mean the column name will be kept for compatibility, but we will take it as `update_time`

Reply:
After changing it to `update_time`, we are unable to display the monitoring panel for `pending duration`. I believe we can still keep the `state_update_time` field and subsequently add a new field called `update_time` to represent the new semantics. This has little cost. It also does not affect other operations.

Reply:
you can cal `pending duration` by now() - create_time` if it's state is pending